lib/CMakeLists.txt

```diff
@@ -112,8 +113,31 @@ else()
 	target_link_libraries(chiaki-lib OpenSSL::Crypto)
 endif()
 
-target_link_libraries(chiaki-lib Nanopb::nanopb)
-target_link_libraries(chiaki-lib Jerasure::Jerasure)
+if(CHIAKI_ENABLE_SWITCH AND NOT CHIAKI_ENABLE_SWITCH_LINUX)
+	# to provides csrngGetRandomBytes
+	target_link_libraries(chiaki-lib nx)
+endif()
+
+#if(CHIAKI_ENABLE_RPI)
+
+# Should set up some auto build of libilclient.
+
+	add_compile_definitions(OMX_SKIP64BIT)
+include_directories(/opt/vc/src/hello_pi/libs/ilclient
```

Comment:
This is a funny one,   /opt/vc/src/hello_pi/libs/ilclient  is installed by default on Raspbian Buster but it is not compiled/built.
There are Makefiles there but I don't remember if I had to slightly edit to avoid running make from the umbrella folder that covers a series of 20+ mini demonstration projects.

Should this be setup to build as well do you think and if so have you got any pointers for how that should be done?
Or should I just add that to a 'RPi Build - How To' ?

Reply:
Those mini projects are also in the RPi 'userland'.
https://github.com/raspberrypi/userland/tree/master/host_applications/linux/apps/hello_pi

Reply:
This should probably be done with a combination of cmake find modules and building it in cmake directly. But I can't say much about the details yet. I would suggest to leave this as-is for now and I will do this building part myself later since it can become quite tricky with cmake.